pkg/aws/ec2/api/wrapper.go

```diff
@@ -789,3 +819,27 @@ func (e *ec2Wrapper) CreateNetworkInterfacePermission(input *ec2.CreateNetworkIn
 
 	return output, err
 }
+
+func (e *ec2Wrapper) GetSecurityGroupsForVpc(input *ec2.GetSecurityGroupsForVpcInput) (*ec2.GetSecurityGroupsForVpcOutput, error) {
+	accountIdFilter := ec2.Filter{
+		Name:   aws.String("owner-id"),
```

Comment:
This can be removed since we already have the `VpcId` filter.

Reply:
Updated